docs/authentication-backends.md

```diff
@@ -85,10 +81,11 @@ Both `git-gateway` and `github` backends allow some additional optional fields f
 cases. A full reference is below. Note that these are properties of the `backend` field, and should
 be nested under that field.
 
-| Field         | Required | Default                                                           | Description                                                                                                                                          |
-|---------------|----------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `repo`        | Yes      | n/a                                                               | The username of the repo owner, and the repo name, separated by a slash.                                                                             |
-| `branch`      | No       | `master`                                                          | The branch to store published content on.                                                                                                            |
-| `api_root`    | No       | `https://api.github.com` (ignored for `git-gateway` backend)      | The API endpoint. Only necessary in certain cases, e.g. for GitHub Enterprise users on the `github` backend.                                         |
-| `site_domain` | No       | `[location].[hostname]` or `cms.netlify.com` when on `localhost`  | Sets the `site_id` query param sent to the API endpoint. Non-Netlify auth setups will often need to set this for local development to work properly. |
-| `base_url`    | No       | `https://api.netlify.com`                                         | OAuth client URL for the `github` backend. Required when using an external OAuth server with the `github` backend.                                   |
\ No newline at end of file
+| Field          | Default                                                           | Description                                                                                                                                          |
+|----------------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `repo`         | none                                                              | **Required** for `github` backend; ignored by `git-gateway`. Follows the pattern `[org-or-username]/[repo-name]`.                                    |
+| `accept_roles` | none                                                              | `git-gateway` only. Limits CMS access to your defined array of user roles. Omitting this field gives access to all registered users.                 |
```

Comment:
I removed the "Required" column because it took up a lot of space, and as it turns out, none of them are required in all cases. Instead, I added "**Required** for x" in the description.
I also added `accept_roles` so we now cover all possible fields (except `name` which felt kind of weird to include—open to adding, though).